### ISSUE TITLE
GameCube/PSX/Logitech controller issue with v2.3.1

### ISSUE THREAD

**Issue by nagra-xii** [OpenEmu/OpenEmu#4405]
Installed the new version of OpenEmu today, on my MacBook Pro running Catalina 10.15.7.

Now all of a sudden, my generic GameCube USB controller doesn't respond at all, nor can I manually programme it; my Logitech Dual Action controller (which typically I use for PSP) half-works--B button is assigned correctly, and the left analogue seems to work, but the A button is assigned to the Y button and I cannot change the control assignments...that's as far into my GameCube game of choice (Eternal Darkness) as I can get.  Similarly, my generic PSX controller isn't working fully with PSX (I can only go forward and backward in Silent Hill) and it does work (but awkwardly) with PSP--yet the Logitech in those contexts (both PSX and PSP) works just fine.  No problem using my generic NES and SNES controllers so far.

With the previous version of OpenEmu, all of my controllers (as well as the roms) responded perfectly and programmed with no issues--so it's not a hardware issue.

Am enjoying the new true fullscreen capability--but the inability to use tried-and-true USB controllers is spoiling the fun.
Thanks so much for any help you can provide.

**Comment by clobber** [comment]
There were no significant changes to the input system in 2.3/2.3.1. While you can replicate the issue, I would immediately switch back to [version 2.2.1](https://github.com/OpenEmu/OpenEmu/releases/download/v2.2.1/OpenEmu_2.2.1.zip) to see if you can replicate it there. If so, sounds like you may have an issue related to something outlined in the [input troubleshooting](https://github.com/OpenEmu/OpenEmu/wiki/Troubleshooting:-Input-problems) guide

**Comment by clobber** [comment]
@nagra-xii any update here?

**Comment by nagra-xii** [comment]
I have no conflicting applications running.  In fact, nothing but my web browser (Opera 73.0.3856.284) was running while I was using OpenEmu.  I read the troubleshooting guide; nothing there applied.  As I said, with the previous version of OpenEmu there were no problems using any of the controllers and all of their input assignments could be manually reconfigured.

**Comment by clobber** [comment]
@nagra-xii Your web browser could be an issue. It's worth restarting your computer to find out if the issue still occurs on OpenEmu 2.3.1 and **IF IT DOES**, immediately try [version 2.2.1](https://github.com/OpenEmu/OpenEmu/releases/download/v2.2.1/OpenEmu_2.2.1.zip) to see if you can replicate the problem there too. I'm not talking about if it *previously* worked here.

**Comment by nagra-xii** [comment]
In the interest of taking your suggestion and restarting the computer, I closed my web browser.  Then I closed OpenEmu.  Just out of curiosity, however, I reopened OpenEmu just to see if anything had changed.  Magically, the more complicated controllers were working just fine and the assignable parameters were functioning as expected.  I reopened my web browser to see if suddenly the conflict would reappear; even with it open, everything worked fine.  Doesn't make a lot of sense as the programme had already been re-launched, but everything works as expected now with PSX, PSP, Gamecube, and their associated controllers.  Thanks for your help.
